build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    kotlin("jvm") version "1.4.0"
+    id("org.jmailen.kotlinter") version "3.0.2"
+    id("com.github.johnrengelman.shadow") version "6.0.0"
+    application
```

Comment:
configures tasks to produce output appropriate for an application, rather than a library (which we actually are disabling below since we want a shaded fat jar), and the `run` task to run the main() method of the class specified below